pyemma/coordinates/api.py

```diff
@@ -1168,10 +1199,95 @@ def tica(data=None, lag=10, dim=-1, var_cutoff=0.95, kinetic_map=True, commute_m
             user_msg,
             category=PyEMMA_DeprecationWarning)
 
-    res = TICA(lag, dim=dim, var_cutoff=var_cutoff, kinetic_map=kinetic_map, commute_map=commute_map, skip=skip)
+    res = TICA(lag, dim=dim, var_cutoff=var_cutoff, kinetic_map=kinetic_map, commute_map=commute_map, skip=skip,
+               weights=weights, reversible=reversible)
     return _param_stage(data, res, stride=stride)
 
 
+def lagged_covariance(data=None, c00=True, c0t=True, ctt=False, remove_constant_mean=None, remove_data_mean=False,
+                      reversible=False, bessel=True, lag=0, reweighting="empirical", stride=1, skip=0, chunksize=None):
```

Comment:
I suggest the name ``covariance_lagged``, because you could also have a ``covariance`` without the lagging.